theme6/source/javascripts/related_products.js

```diff
@@ -0,0 +1,27 @@
+
+var images = ['bear_toy', 'train_toy', 'plane_toy', 'ship_toy'];
+var i = 0;
+
+function changeImage(value) {
+  i += value;
+
+  if (i < 0) {
+    i = images.length - 1;
+  }
+  else if (i > images.length - 1) {
+    i = 0;
+  }
+
+  document.getElementById('featuredToy').src = 'images/' + images[i] + '.png';
+}
+
+const leftArrow = document.getElementById('leftArrow');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).